pkg/user/tx_client.go

```diff
@@ -242,7 +242,10 @@ func SetupTxClient(
 		}
 		accNum, seqNum, err := QueryAccount(ctx, conn, encCfg.InterfaceRegistry, addr)
 		if err != nil {
-			// skip over the accounts that don't exist in state
+			// Instead of skipping, add the account with zero values
+			// This allows the account to be added to the Signer's account map
+			// and potentially be lazy-loaded later when needed
+			accounts = append(accounts, NewAccount(record.Name, 0, 0))
```

Comment:
IMO, I don't think we should add partially filled accounts here. It just requires too much additional logic. Either an account is full added (with account number and sequence number) or not at all (and thus can be lazily loaded during `checkAccountLoaded`)